### PR TITLE
Sb/7

### PR DESCRIPTION
**Pull request by kilmbolden760**

### CODE REVIEW

**Comment by kilmbolden760** [comment]
Hope this worked.

File: shaquille/stocks_challenge.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+amazon = 3000
+apple = 100
+facebook = 250
+google = 1400
+microsoft = 200
+
+print("Challenge 3.2.1: Taking user input")
+# TODO: Write code to ask the client his name and save it to a variable.
+Name= input("what is your name? ")
```
Comment (liubrandon) [inline shaquille/stocks_challenge.py:13]: Variables should start with lowercase letters

File: shaquille/stocks_challenge.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+elif number_of_stocks>0:
+    print(f"Congratulations {Name} you have {Savings} of savings and you can buy {number_of_stocks} Amazon stocks ")
+elif number_of_stocks<0:
+    print("Sorry, you can not buy Amazon stocks with the {savings}")
+elif stock== aapl:
+     number_of_stocks=(Savings/apple)
+elif number_of_stocks>0:
+    print(f"Congratulations {Name} you have {Savings} of savings and you can buy {number_of_stocks} Apple stocks ")
+elif number_of_stocks<0:
+    print("Sorry, you can not buy Apple stocks with the {savings}")
```
Comment (liubrandon) [inline shaquille/stocks_challenge.py:41]: These elifs are duplicated, you only need them once

File: shaquille/stocks_challenge.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+number_of_stocks= 0
+
+if stock == amzn:
+    number_of_stocks=(Savings/amazon)
+elif number_of_stocks>0:
+    print(f"Congratulations {Name} you have {Savings} of savings and you can buy {number_of_stocks} Amazon stocks ")
+elif number_of_stocks<0:
+    print("Sorry, you can not buy Amazon stocks with the {savings}")
+elif stock== aapl:
```
Comment (liubrandon) [inline shaquille/stocks_challenge.py:36]: I think you need 'aapl' (and the subsequent ones) in quotes. Right now you're checking if stock is equal to the aapl variable which you set to apple which is 100

File: shaquille/stocks_challenge.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+
+print("Challenge 3.2.3: Output for the user the result")
+# TODO: Once you have calculated the number of stocks that can be purchased, print the result for the client. Result should be in a format like this:
+
+# Alex has $5000 in savings and he can buy 50 shares of Apple at the current price of $100.
+
+
+
+print(f"{Name} has {Savings} and he can buy {number_of_stocks} shares of {stock} at the current price of $100")
```
Comment (liubrandon) [inline shaquille/stocks_challenge.py:71]: Not all of the stocks are $100, you should print the corresponding price

File: shaquille/snippets_challenge.py [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+print("Code Snippet 4:")
+
+#modify exactly one boolean operator in the assignment of d, so that d evaluates to False
+
+d = (5 > 7) or not (8 < 20)
+# TO DO: Explain how d is set to False in a print statement
+assert(d == False) #Do not change this line
+
+print(f"the value of ({d}) is False")
```
Comment (liubrandon) [inline shaquille/snippets_challenge.py:55]: You didn't explain why as asked above

**Comment by liubrandon** [comment]
In the future, new challenges (buy_me_a_laptop.py) should be in their own branch. left some comments and will send you solutions!

**Review by liubrandon — APPROVED** [review]

**Pull request merged**